src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesConfigure/FeatureStrategiesConfigure.tsx

```diff
@@ -15,13 +15,16 @@ import { IFeatureViewParams } from '../../../../../../interfaces/params';
 import cloneDeep from 'lodash.clonedeep';
 import FeatureStrategyCreateExecution from '../../FeatureStrategyCreateExecution/FeatureStrategyCreateExecution';
 import { PRODUCTION } from '../../../../../../constants/environmentTypes';
+import { ADD_NEW_STRATEGY_SAVE_ID } from '../../../../../../testIds';
```

Comment:
not important for now;
but i think we can configure our module resolving to support `lib` like this instead:

`import { ADD_NEW_STRATEGY_SAVE_ID } from 'lib/testIds'`;

Reply:
https://www.typescriptlang.org/docs/handbook/module-resolution.html

Reply:
https://decembersoft.com/posts/say-goodbye-to-relative-paths-in-typescript-imports/